go/enclave/rollupchain/rollup_chain.go

```diff
@@ -98,33 +98,6 @@ func New(
 	}
 }
 
-// ProduceNewRollup creates a new rollup, building on the latest chain heads.
-func (rc *RollupChain) ProduceNewRollup() (*common.ExtRollup, error) {
```

Comment:
This function was moved down, since it is now internal (called as part of SubmitL1Block).